dags/check_brokenreaders.py

```diff
@@ -0,0 +1,81 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.operators.python_operator import PythonOperator
+from airflow.hooks.postgres_hook import PostgresHook
+from airflow.hooks.base_hook import BaseHook
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+from psycopg2 import sql
+from psycopg2.extras import execute_values
+from psycopg2 import connect, Error
+import logging
+import configparser
+
+LOGGER = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+#To connect to pgadmin
+CONFIG = configparser.ConfigParser()
```

Comment:
I should set up a bt bot account for Airflow so then the credentials would be imported 
https://github.com/CityofToronto/bdit_data-sources/blob/master/dags/pull_wys.py#L59-L60